package.json

```diff
@@ -97,6 +97,7 @@
   },
   "devDependencies": {
     "@types/classnames": "^2.2.4",
+    "@types/enzyme": "^3.1.14",
```

Comment:
Is this intentional?

Reply:
yes, these typings are very useful in the tests for autocomplete the API suggestions